packages/tiny-agents/src/index.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
Probably some code should be added for programmatic usage?

Reply:
1c0215e1bdca5ca68c13708c36e89ac88e1a0630